src/tools/Makefile.am

```diff
@@ -178,5 +179,11 @@ gids_tool_SOURCES += versioninfo-tools.rc
 opensc_notify_SOURCES += versioninfo-opensc-notify.rc
 endif
 
+applicationsdir = $(datadir)/applications
+applications_DATA = org.opensc-project.opensc-notify.desktop
+
+org.opensc-project.opensc-notify.desktop: org.opensc-project.opensc-notify.desktop
```

Comment:
This should say (adding `.in` to the end):
```
org.opensc-project.opensc-notify.desktop: org.opensc-project.opensc-notify.desktop.in
```

Reply:
done, thanks